spec/cli_spec.rb

```diff
@@ -4,7 +4,35 @@
   context '#run_suite' do
     it 'calls spec runner' do
       expect_any_instance_of(FailSpell::SpecRunner).to receive(:run_and_store_results)
-      FailSpell::CLI.new.run_suite
+      described_class.new.run_suite
+    end
+  end
+
+  context '#rerun_failed' do
+    it 'updated successful tests in json file' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.